libraries/cmake/source/grpc/cmake/protobuf.cmake

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) 2014-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed in accordance with the terms specified in
+# the LICENSE file found in the root directory of this source tree.
+
+function(protobufMain)
+  set(library_root "${CMAKE_CURRENT_SOURCE_DIR}/src/third_party/protobuf/")
+
+  add_library(thirdparty_protobuf
+    # libprotobuf_lite
+    ${library_root}/src/google/protobuf/any_lite.cc
+    ${library_root}/src/google/protobuf/arena.cc
+    ${library_root}/src/google/protobuf/arenastring.cc
+    ${library_root}/src/google/protobuf/arenaz_sampler.cc
+    ${library_root}/src/google/protobuf/extension_set.cc
+    #${library_root}/src/google/protobuf/generated_enum_util.cc
```

Comment:
can this be removed?

Reply:
Yes, not sure why that was left in, I'll remove it.